extras/sht3x/README.md

```diff
@@ -0,0 +1,256 @@
+# Driver for **SHT3x** digital **temperature and humidity sensor**
+
+This driver is written for usage with the ESP8266 and FreeRTOS using the I2C interface driver ([esp-open-rtos](https://github.com/SuperHouse/esp-open-rtos) and [esp-open-rtos-driver-i2c](https://github.com/kanflo/esp-open-rtos-driver-i2c)).
```

Comment:
It does not appear necessary to reference the esp-open-rtos i2c driver source code like this, and this driver is not even using the kanflo version referenced.

Reply:
Removed it.